src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -62,16 +66,11 @@ public class HubEndPoint<THub, TClient> : EndPoint, IInvocationBinder where THub
                 await _lifetimeManager.OnConnectedAsync(connection);
 
                 using (var scope = _serviceScopeFactory.CreateScope())
+                using (var hubWrapper = _hubActivator.Create<THub, TClient>(scope.ServiceProvider))
```

Comment:
This doesn't look right. Why do we need to pass the service provider? We should be looking at MVC and copying their patterns where possible.